aspnetcore/blazor/tutorials/movie-database-app/part-7.md

```diff
@@ -0,0 +1,382 @@
+    name: "Rating",
+    table: "Movie",
+    type: "nvarchar(max)",
+    nullable: false,
+-   defaultValue: "");
++   defaultValue: "R");
+```
+
+Save the migration file.
+
+In the **SQL Server Express LocalDB** area of **Service Dependencies**, select the ellipsis (`...`) again followed by the **Update database** command.
+
+The **Update database with the latest migration** dialog opens. Wait for the **DbContext class names** field to update and for prior migrations to load. Select the **Finish** button.
+
+Modify the one movie that isn't rated *R*:
+
+1. Run the app.
+1. Edit the *Mad Max: Beyond Thunderdome* movie.
+1. Update the movie rating from `R` to `PG-13`. Save the change.
+
+> [!NOTE]
+> An alternative to modifying the migration file is to delete the records in the database and rerun the app to reseed the database. The seeding code was modified earlier to supply default values. This approach is useful in cases where the assignment of default values to fields is better controlled or faster for you to implement with C# code during seeding.
+>
+> To delete all of the records in the database, use one of the following approaches:
+>
+> * Run the app and use the delete links in the browser. This approach is reasonably fast when there are only a few records to delete.
+> * In Visual Studio from **SQL Server Object Explorer** (SSOX), delete the database records. With the database table visible, right-click the table and select **View Data**. When the table opens to show the movie records, select the first record. Hold the <kbd>Shift</kbd> key and select the last record to select all of the records in the table. With all of the records selected, press the <kbd>Delete</kbd> key to delete the records. This approach is reasonably fast when there are many records to delete.
+> * In Visual Studio from SSOX, right-click the database and select **New Query**. A blank SQL file (`.sql`) opens. Paste the following command into the file: `DELETE FROM dbo.Movie;`. Select the green **Execute** triangle to execute the query or press <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>E</kbd> on the keyboard. In the **Message** tab, SSOX reports the number of rows affected, deleted in this case, by the query. You can save the query for later use if you wish. Otherwise, close the file without saving it. This approach is especially useful when the table is large and contains hundreds to thousands of records.
+>
+> **CAUTION**: *Always use extreme caution when deleting records. Deleting records is permanent without taking additional data loss mitigation steps. Production databases often provision automatic backup copies of data, either instantaneously as the database is modified or periodically, including with off-site copies and permanent physical storage of data.*
+>
+> After deleting all of the records, run the app. The initializer reseeds the database and includes the correct movie ratings for the `Rating` field based on the seeding code.
```

Comment:
This is a large Note with an embedded Caution warning inside of it. Would it be better to just make the Note part of the text and making the Caution warning an actual warning?